tests/core/reproducibility/test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -eu -o pipefail
+
+function run_bazel() {
+    local output_base
+    local bazel_opts
+    local build_opts
+
+    output_base=$(mktemp -d)
+    bazel_opts=(
+    "--bazelrc=/dev/null"  # Disable any user specified remote/disk caching.
+    "--output_base=${output_base}"  # Use a disposable output_base.
+    )
+    build_opts=()
+
+    if [[ $(uname -s) == "Linux" ]]; then
+      # Use shared memory sandbox for faster runs.
+      build_opts+=("--experimental_sandbox_base=/dev/shm")
+    fi
+
+    bazel "${bazel_opts[@]}" build "${build_opts[@]+"${build_opts[@]}"}" \
+      //tests/core/go_binary:all \
```

Comment:
It would be better to test against targets in this directory instead of elsewhere. `//tests/core/go_binary:all` may pick up some unexpected things, depending on what we put in there in the future. It's better for this test to be isolated.

Reply:
Alright. I will create some binary with cgo here.

Reply:
Done.